### PR TITLE
Simplify the API baseline target

### PR DESCRIPTION
**Pull request by merks**
Also simplify the corresponding API baseline targlet in the setup

https://github.com/eclipse-cdt/cdt/issues/39

### CODE REVIEW

**Comment by merks** [comment]
@jonahgraham @ruspl-afed 

Guys, I submitted the changes as PR.   Then I can see how the API baseline is used in the build and you can more easily review this in your local workspace...

File: releng/org.eclipse.cdt.target/cdt-baseline.target [hunk 1]
```diff
@@ -1,101 +1,47 @@
...
+      <unit id="org.eclipse.cdt.visualizer.feature.group" version="0.0.0"/>
+      <unit id="org.eclipse.launchbar.remote.feature.group" version="0.0.0"/>
+      <unit id="org.eclipse.remote.console.feature.group" version="0.0.0"/>
+      <unit id="org.eclipse.remote.proxy.feature.group" version="0.0.0"/>
+      <unit id="org.eclipse.remote.serial.feature.group" version="0.0.0"/>
+      <unit id="org.eclipse.remote.telnet.feature.group" version="0.0.0"/>
+      <unit id="org.eclipse.tm.terminal.connector.cdtserial.feature.feature.group" version="0.0.0"/>
+      <unit id="org.eclipse.tm.terminal.connector.remote.feature.feature.group" version="0.0.0"/>
+      <unit id="org.eclipse.tm.terminal.feature.feature.group" version="0.0.0"/>
+      <repository location="https://download.eclipse.org/tools/cdt/releases/latest"/>
```
Comment (ruspl-afed) [inline releng/org.eclipse.cdt.target/cdt-baseline.target:44]: Usually we have repository at the top and then we sort locations by repository::location

**Review by ruspl-afed — APPROVED** [review]
I greet this change as it reduces support effort. Only one minor comment

**Comment by merks** [comment]
I kind of doubt the test failures can be caused the API baseline change...

Now that I look closely, I don't think the baseline target is even used.  The actual Maven build uses this which is out of date too:
```
		<comparator.repo>https://download.eclipse.org/tools/cdt/releases/10.6/cdt-10.6.0/</comparator.repo>
```

**Comment by merks** [comment]
Given the cdt-baseline.target is not used in the build, but rather is only used for a manual setup, reducing maintenance effort would definitely be a good thing...

**Comment by jonahgraham** [comment]
> I kind of doubt the test failures can be caused the API baseline change...
> 
> Now that I look closely, I don't think the baseline target is even used. The actual Maven build uses this which is out of date too:
> 
> ```
> 		<comparator.repo>https://download.eclipse.org/tools/cdt/releases/10.6/cdt-10.6.0/</comparator.repo>
> ```

That is the compare and replace repo. Separate from api baseline. As mentioned in the other comment I just made that https://github.com/eclipse-cdt/cdt-infra/issues/59 needs to be done for next version and currently main hasn’t been prepped for post 10.7 yet. 

BTW main branch will be 11.0 next, that is a major release of CDT is upcoming.

**Comment by jonahgraham** [comment]
I’m glad that you (Ed) are simplifying and that Alexander has time to review these changes. I will happily review anything left to review later.

**Pull request merged**